python/templates/datamodel.h.jinja2

```diff
@@ -12,21 +12,27 @@
 namespace {{ package_name }} {
 
 using {{ package_name }}DataTypes = podio::utils::TypeList<
-{% for name in datatypes %}
-  {{ name }}{% if not loop.last %},{% endif %}
-{% endfor %}
+  {{ datatypes | join(', ')}}
+>;
+
+using {{ package_name }}DataCollectionTypes = podio::utils::TypeList<
```

Comment:
```suggestion
using {{ package_name }}CollectionTypes = podio::utils::TypeList<
```
Just realized that this should have been without the `Data` here. Currently, it reads `edm4hepDataCollectionTypes` for example.